api/loader/getter.go

```diff
@@ -71,17 +71,21 @@ func getRemoteTarget(rs *remoteTargetSpec) error {
 		log.Fatalf("Error getting wd: %s", err)
 	}
 
+	httpGetter := &getter.HttpGetter{
+		Netrc: true,
+	}
+
 	opts := []getter.ClientOption{}
 	client := &getter.Client{
 		Ctx:  context.TODO(),
 		Src:  rs.Raw,
 		Dst:  rs.Dir.String(),
 		Pwd:  pwd,
-		Mode: getter.ClientModeAny,
-		Detectors: []getter.Detector{
-			new(getter.GitHubDetector),
-			new(getter.GitDetector),
-			new(getter.BitBucketDetector),
+		Mode: getter.ClientModeDir,
```

Comment:
Remote bases must be directories containing a `kustomization.yaml` file so there's no sense in fetching a single file.